Dockerfile

```diff
@@ -13,11 +23,11 @@ COPY --chown=1001:101 $APP_PATH /app/samvera/hyrax-webapp
 ARG HYKU_BULKRAX_ENABLED="true"
 RUN sh -l -c " \
   yarn install && \
-  RAILS_ENV=production SECRET_KEY_BASE=FAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKE DB_ADAPTER=nulldb DATABASE_URL='postgresql://fake' bundle exec rake assets:precompile"
+  RAILS_ENV=production SECRET_KEY_BASE=FAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKEFAKE DB_ADAPTER=nulldb DB_URL='postgresql://fake' bundle exec rake assets:precompile"
 RUN ln -sf /app/samvera/branding /app/samvera/hyrax-webapp/public/branding
 
-CMD ./bin/web
```

Comment:
If this `CMD` is removed, how does the rails server start in the `web` container?

Reply:
Well…how did UTK do it?  (I'm restoring this)

Reply:
It's possible it comes from this line? 

```
FROM ghcr.io/scientist-softserv/dev-ops/samvera:f71b284f as hyku-base
```

It's possible there's a `CMD` line in that file that gets inherited. 

If the server started without it I think it's fine, I was mostly asking out of curiosity (my other comment was the true "requested change")